Makefile

```diff
@@ -73,14 +73,15 @@ IMAGE_BUILD_ARGS = --build-arg VERSION=$(VERSION) \
                 --build-arg BASE_IMAGE_MINIMAL=$(BASE_IMAGE_MINIMAL)
 
 IMAGE_BUILD_ARGS_FULL = --target full \
-                	-t $(IMAGE_TAG) \
-	    		$(foreach tag,$(IMAGE_EXTRA_TAGS),-t $(tag)) \
-	    		$(IMAGE_BUILD_EXTRA_OPTS) ./
+                        -t $(IMAGE_TAG) \
+                        $(foreach tag,$(IMAGE_EXTRA_TAGS),-t $(tag)-full) \
+                        $(IMAGE_BUILD_EXTRA_OPTS) ./
 
 IMAGE_BUILD_ARGS_MINIMAL = --target minimal \
-                	   -t $(IMAGE_TAG)-minimal \
-	            	   $(foreach tag,$(IMAGE_EXTRA_TAGS),-t $(tag)-minimal) \
-	            	   $(IMAGE_BUILD_EXTRA_OPTS) ./
+                           -t $(IMAGE_TAG) \
+                           -t $(IMAGE_TAG)-minimal \
+                           $(foreach tag,$(IMAGE_EXTRA_TAGS),-t $(tag) -t $(tag)-minimal) \
+                           $(IMAGE_BUILD_EXTRA_OPTS) ./
```

Comment:
```suggestion
                           -t $(IMAGE_TAG)-minimal \
                           $(foreach tag,$(IMAGE_EXTRA_TAGS),-t $(tag)-minimal) \
                           $(IMAGE_BUILD_EXTRA_OPTS) ./
```

Reply:
We still definitely want to keep the "default" image tag (i.e. just `v0.13.0` without any `-minimal` or `-full` suffix). As explained in the commit message this PR keeps the `-minimal` tag for backwards compatibility. Makes sense?

Reply:
ah right.